docs/api-testing/use-cases/saving-token-kv.md

```diff
@@ -0,0 +1,278 @@
+
+   This step calls the endpoint that generates the token.
+
+7. **Save Changes**.
+
+8. **Add Child Component** for the **If** and then, add the **Key/Value Store** component.
+
+   - Action - for example `Set`
+   - Key - for example `token`
+   - Data - for example `payload.token`
+
+   <img src={useBaseUrl('img/api-testing/kv-examples/token-kv2.png')} alt="set method in key value component" />
+
+   In this step you add the value of the token in the Key/Value store.
+
+9. **Save Changes**.
+
+10. **Add Child Component** inside the **If** and then, add the **Key/Value Store** component.
+
+    - Action - for example `Set`
+    - Key - for example `last_token`
+    - Data - for example `D.nowMillis()`
+
+    <img src={useBaseUrl('img/api-testing/kv-examples/refresh-kv2.png')} alt="set method in key value component" />
+
+    In this step you add the date in the Key/Value Store.
+
+11. **Save Changes**.
+
+12. Outside the **If**, add **Assert Exists**.
+
+    - Expression - for example `token_var`
+
+    <img src={useBaseUrl('img/api-testing/kv-examples/token-assert.png')} alt="assert exists" />
+
+    The token will not be generated again for the next minute and the existing one will be used.
+
+13. Add the **POST** component and then, **Save Changes**.
+
+- Url - for example `https://m2-keyvalueexample.load2.apifortress.com/profile`
+- Variable - for example `profilePayload`
+- Mode - for example `json`
+
+<img src={useBaseUrl('img/api-testing/kv-examples/token-post.png')} alt="POST request" />
+
+This step calls the endpoint that requires the token for authenticate.
+
+14. Add the **Request Header** to the **POST**.
+
+    - Name - for example `Authorization`
+    - Value - for example `${token_var}`
+
+    <img src={useBaseUrl('img/api-testing/kv-examples/token-auth.png')} alt="Request Header" />
+
+    In the request header you pass the token saved in the Key/Value Store.
+
+The test looks like:
+
+```yaml
+- id: kv
+  key: token
+  action: load
+  var: token_var
+- id: kv
+  key: last_token
+  action: load
+  var: last_token_var
+- id: if
+  children:
+    - id: get
+      children: []
+      url: https://m2-keyvalueexample.load2.apifortress.com/token
+      var: payload
+      mode: json
+    - id: kv
+      key: token
+      action: set
+      object: payload.token
+    - id: kv
+      key: last_token
+      action: set
+      object: D.nowMillis()
+  expression: "!token_var || ! last_token_var || last_token_var+60000 < D.nowMillis()"
+- id: assert-exists
+  expression: token_var
+- id: post
+  children:
+    - id: header
+      name: Authorization
+      value: ${token_var}
+  url: https://m2-keyvalueexample.load2.apifortress.com/profile
+  var: profilePayload
+  mode: json
+```
+
+By running the test two times in a row (in less then 1 minute) you can notice that the first time, the test will execute the call that generates the token and the second time it will execute only the call that uses the token.
```

Comment:
```suggestion
By running the test two times in a row (in less than one minute) you can notice that the first time, the test will execute the call that generates the token and the second time it will execute only the call that uses the token.
```